cfme/tests/containers/test_properties.py

```diff
@@ -132,8 +134,7 @@
 ]
 
 
-@pytest.mark.parametrize('test_item', TEST_ITEMS,
-                         ids=[ti.args[1].pretty_id() for ti in TEST_ITEMS])
+@pytest.mark.parametrize('test_item', TEST_ITEMS)
```

Comment:
I need to have something that is either a string, or has a .name attribute for a parameter else it will screw up future reporting.